locale/core.yml

```diff
@@ -11,7 +11,7 @@ core:
     add_extension:
       developer_text: "Si vous êtes un développeur, vous pouvez <a>consultez la documentation</a> pour vous initier à créer vos propres extensions."
       install_text: "De même, vous pouvez rechercher des extensions sur la <a>communauté officielle de Flarum</a> et les installer manuellement en utilisant Composer."
-      temporary_text: "Prochainement, cette fenêtre vous permettra d’ajouter en quelques clics une extension à votre forum. Nous sommes d’ailleurs en train de développer cet écosystème !"
+      temporary_text: "Prochainement, cette fenêtre vous permettra d’ajouter en quelques clics une extension à votre forum. Nous sommes d’ailleurs en train de développer cet écosystème !"
```

Comment:
I'm not sure what's wrong with this changed spaces either.

Reply:
Some spaces are nonbreaking spaces (it's a French rule, these spaces are before and/or after some characters, like ! ? : etc. They should stay like that.

Plus, this PR don't seems to fix the issue. Thanks anyway! 😎

Reply:
yes, I've reverted those, but according to travis-ci that error gone

Reply:
Right! I'm not at home right now, so I can't merge it for now. Thanks a lot sijad!